test/cpp/jit/test_flatbuffer.cpp

```diff
@@ -1135,5 +1135,91 @@ TEST(FlatbufferTest, OperatorTest2) { // NOLINT (use =delete in gtest)
   }
 }
 
+Module jitModuleFromBuffer(void* data) {
```

Comment:
Would it be parse_and_initialize_jit_module() below?

Reply:
yeah though if doing so would need to transfer ownership from flatbuffers::DetachedBuffer to mobile::Module so it's bit annoying.